app/src/main/java/org/mozilla/focus/widget/DownloadListAdapter.java

```diff
@@ -78,6 +78,9 @@ private void delete(int position){
         }catch (Exception e){
             Log.v(this.getClass().getSimpleName(),""+e.getMessage());
         }
+
```

Comment:
Please submit different pull requests for different bugs so I can merge/review each bug separately. The first two commits are good. (Typo in the first commit, though.)